dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -102,32 +131,107 @@ public CompletableResultCode export(@Nonnull Collection<MetricData> metrics) {
     return export(metrics, connection);
   }
 
+  private List<String> makeMetricLines(Collection<MetricData> metrics) {
+    ArrayList<String> metricLines = new ArrayList<>();
+    for (MetricData metric : metrics) {
+      boolean isDelta;
+      switch (metric.getType()) {
+        case LONG_GAUGE:
+          metricLines.addAll(serializer.createLongGaugeLines(metric));
+          break;
+        case LONG_SUM:
+          isDelta =
+              metric.getLongSumData().getAggregationTemporality() == AggregationTemporality.DELTA;
+          metricLines.addAll(serializer.createLongSumLines(metric, isDelta));
+          break;
+        case DOUBLE_GAUGE:
+          metricLines.addAll(serializer.createDoubleGaugeLines(metric));
+          break;
+        case DOUBLE_SUM:
+          isDelta =
+              metric.getDoubleSumData().getAggregationTemporality() == AggregationTemporality.DELTA;
+          metricLines.addAll(serializer.createDoubleSumLines(metric, isDelta));
+          break;
+        case SUMMARY:
+          metricLines.addAll(serializer.createDoubleSummaryLines(metric));
+          break;
+        case HISTOGRAM:
+          metricLines.addAll(serializer.createDoubleHistogramLines(metric));
+          break;
+      }
```

Comment:
Please add a default case that warns on unhandled types (in case someone bumped the upstream dependency on their own).